src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    int index = new Random().nextInt(Color.values().length);
```

Comment:
introduce `index` as local variable but `Random` instance as field variable. Your `index` now is not random when you call `getRandomColor`.